test/Rings-conformance-tests.jl

```diff
@@ -210,6 +210,10 @@ function test_Ring_interface(R::AbstractAlgebra.Ring; reps = 50)
             B = deepcopy(b)
             @test isone(inv(one(R)))
             @test a*b == b*a
```

Comment:
```suggestion
            @test a*b == b*a
            @test equality(a^0, one(R))
```
Is this controversial?

Reply:
I think it is, so i left it out.